### DISCUSSION TITLE
Animatediff SDXL support?

### DISCUSSION THREAD

**Discussion by Soulreaver90** [continue-revolution/sd-webui-animatediff#529] [General]
Saw this released and wondering if this will lead to official animatediff support for SDXL?

https://github.com/huggingface/diffusers/releases/tag/v0.28.0

**Comment by NergiZed** [comment]
I hope so. Although there are SDXL motion models for Animatediff right now. They don't work as well as the SD 1.5 ones though.

**Comment by continue-revolution** [comment]
You can use that sdxl beta checkpoint in this extension, but unfortunately the performance is not satisfactory

**Reply by Soulreaver90** [reply]
Really? I was under the impression it didn’t work as it throws an error anytime I use it. I’ll try to clone a fresh install and try again. If I get the same error, I’ll post a copy or open an issue.
